src/components/Collapsible.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+
+export default function Collapsible ({ children }) {
+    const [isHidden, setIsHidden] = useState(false);
+    const toggleIsHidden = () => {
+        setIsHidden(!isHidden);
+    };
+    return (
+        <div className='collapsibleDiv'>
+            <button onClick={toggleIsHidden}>
+                {`${isHidden ? 'Show Comments' : 'Hide Comments'}`}
+            </button>
+            {isHidden ? null : children}
+        </div>
```

Comment:
Please ensure button text (and generally any text) is readable. This is being displayed as black text on a dark background